components/StatusItem/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import Icon from './../Icon';
+
+const StatusItemIcon = ({ danceStatus }) => {
+  if (danceStatus === 'Uploaded') {
```

Comment:
Instead of hardcoding each string for each case, can you create a constant in this file:

```
const DANCE_STATUS = {
  uploaded: 0,
  loading: 1,
  requiredConnection: 2,
}
```

Which can then be exported and imported when we want to use it (e.g. index.test.js below)